client/src/Pages/Loading.tsx

```diff
@@ -7,8 +7,8 @@ import { Oval } from 'react-loader-spinner';
 const Loading = () => {
   const navigate = useNavigate();
   const currUser = useAppSelector((state) => state.userProfile.value);
-  if (currUser.name || currUser.username) {
-    navigate('/');
+  if (currUser.name) {
+    navigate('/landingPage');
```

Comment:
Need both of these, for local auth. Should look like this 

``
if (currUser.name || currUser.username) {
    navigate('/landingPage');
}
``

Reply:
Gotcha